Modules/_testcapimodule.c

```diff
@@ -6186,6 +6206,7 @@ static PyMethodDef TestMethods[] = {
     {"pyobject_fastcall", test_pyobject_fastcall, METH_VARARGS},
     {"pyobject_fastcalldict", test_pyobject_fastcalldict, METH_VARARGS},
     {"pyobject_vectorcall", test_pyobject_vectorcall, METH_VARARGS},
+    {"pyobject_setvectorcall", test_pyobject_setvectorcall, METH_VARARGS},
```

Comment:
The names should match, and this only applies to functions. I'd suggest `function_setvectorcall`